shell/src/main/java/tachyon/shell/command/TouchCommand.java

```diff
@@ -29,6 +29,12 @@
  */
 public final class TouchCommand extends AbstractTfsShellCommand {
 
+  /**
+   * Constructs a new instance to create a 0 byte file specified by argv.
```

Comment:
I feel like there is no need to duplicate the class's purpose in the constructor comment, it's enough to just put it in class comment.